src/Tickets/Commerce/Order.php

```diff
@@ -28,6 +37,35 @@ class Order extends Abstract_Order {
 	 */
 	const POSTTYPE = 'tec_tc_order';
 
+	/**
+	 * Meta key for the checkout status of the order.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected const CHECKOUT_COMPLETED_META = '_tec_tc_checkout_completed';
+
+	/**
+	 * Meta key for the order lock status.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected const ORDER_LOCK_KEY = 'post_content_filtered';
```

Comment:
If we want to hijack a particular column, I would call it a column instead of Key, which kind alludes to Meta Keys.

I would rather hijack `ping_status` which is a `varchar(20)`, which can speed queries depending on the MySQL engine being used, since `post_content_filtered` is a `longtext`.

Reply:
the key is 34 chars - considering for mb4 we need at least varchar( 132 )

Reply:
`post_password`?

Reply:
ill try it but i dont like it because it is actually being used by WP. Dont want to end up facing weird uncalculated issues.